api/api.go

```diff
@@ -26,7 +26,7 @@ var (
 	ClientConnTimeout = 1 * time.Second
 )
 
-// QueryOptions are used to parameterize a query
+// QueryOptions are used to parametrize a query
```

Comment:
TIL

Reply:
it's one of those i say out-loud when i'm typing.